src/js/components/put-away/PutAwaySecondPage.jsx

```diff
@@ -29,20 +29,19 @@ const SelectTreeTable = (customTreeTableHOC(ReactTable));
 class PutAwaySecondPage extends Component {
   constructor(props) {
     super(props);
-    const {
-      putAway, pivotBy, expanded, location,
```

Comment:
`const { initialValues: { putAway, pivotBy, expanded, location } } = this.props;`